README.md

```diff
@@ -31,13 +31,15 @@ You can initilize the client by passing a hash to it, or by passing a block.
 
 ```ruby
 client = Bambora::Client.new(
+  base_url: ENV.fetch('BAMBORA_BASE_URL') # Sandbox or Production URL
```

Comment:
Now a `base_url` is required, and is passed explicitly so that the user can choose either production or sandbox.